### PR TITLE
DOCS-1294: Remove duplication in item description

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/api/payment_methods/post_order_afterpay.md [hunk 1]
```diff
@@ -344,36 +344,36 @@ The amount (in cents) that the customer needs to pay.
 ----------------
 __description__ | string
 
-A text which will be shown with the order in MultiSafepay Control. If the customer's bank supports it this description will also be shown on the customer's bank statement. Max. 200 characters. HTML is not supported. Use the 'items' or 'shopping_cart' objects for this.
+Text displayed with the order in your MultiSafepay Control, and on the customer's bank statement (if supported by the bank). Max. 200 characters. HTML is not supported. Use the `items` or `shopping_cart` objects.
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post_order_afterpay.md:347]:
```suggestion
Text displayed with the order in your MultiSafepay Control, and on the customer's bank statement (if supported by the bank). Max. 200 characters. HTML formatting is not supported. To add descriptions in HTML format, use the `items` or `shopping_cart` objects instead.
```

File: content/api/payment_methods/post_order_afterpay.md [hunk 2]
```diff
@@ -228,35 +228,35 @@ The amount (in cents) that the customer needs to pay.
...
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post_order_afterpay.md:231]:
```suggestion
Text displayed with the order in your MultiSafepay Control, and on the customer's bank statement (if supported by the bank). Max. 200 characters. HTML formatting is not supported. To add descriptions in HTML format, use the `items` or `shopping_cart` objects instead.
```

**Review by Kris-MultiSafepay — CHANGES_REQUESTED** [review]
Good changes, but I don't see an implementation of Ross' suggestions

**Comment by FrancescaNCook** [comment]
Ohhhh I see, not sure how i muddled them (Jira linked tickets probably), but changes only relate to DOCS-2209 not DOCS-1294.

**Review by Kris-MultiSafepay — APPROVED** [review]

**Pull request merged**